int PWM
int CurtisDirection
int ServoAngle
int ServoDirection
int ServoPower
int Watchdog
bool ResetBackToAutoMode
bool OperatorLoggedIn
